Marlin/Marlin_main.cpp

```diff
@@ -1691,6 +1691,11 @@ static void do_blocking_move_to(float x, float y, float z, float feed_rate = 0.0
   feedrate = old_feedrate;
 }
 
+inline void do_blocking_move_to_axis(AxisEnum axis, float where, float feed_rate = 0.0) {
```

Comment:
Isn't the "axis moving to" ?

What is the difference between `do_blocking_axis_move_to()` and `do_blocking_move_to()` ? You want to have a single function that allows you to `move axis to` a coordinate vs the need to call multiple functions, each one for each axis.

From a semantics standpoint the correct term is what Scott said "do_blocking_move_axis_to", from a "sorting/grouping" standpoint yours could make sense.